arbos/block_processor.go

```diff
@@ -422,6 +424,10 @@ func ProduceBlockAdvanced(
 			blockGasLeft = 0
 		}
```

Comment:
Unrelated to your change, but since you're touching this file maybe you could address this as well. Looks like this reimplements logic of SaturatingUSub. I think both of these are uints so we can just use:
blockGasLeft = arbmath.SaturatingUSub(blockGasLeft, computeUsed)

Reply:
I noticed this in a couple of places too, I'll make similar updates wherever possible.
Thanks for pointing this out!